.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+on:
+  push:
+  release:
+    types: [published]
+
+jobs:
+  build_images:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up Python"
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
+
+      - name: "python backend - Running unit tests"
+        working-directory: srv/python
+        run: |
+          pip install -r requirements.txt -r dev-requirements.txt
+          pip install -e .
+          pytest mviewerstudio_backend/test.py
+
+      - name: "python backend - Typechecking"
+        working-directory: srv/python
+        run: |
+          mypy --ignore-missing mviewerstudio_backend
```

Comment:
Can you add a `black` check ? Something like this

```
- name: "check formating"
  working-directory: srv/python
  run: |
    black . --check
```

Reply:
done